com_zimbra_attachmentalert/com_zimbra_attachmentalert.properties

```diff
@@ -19,7 +19,7 @@ description = Alerts you when forgetting to attach an attachment when composing
 #
 # Defines the attach search strings (a comma separated list)
 #
-AttachmentAlert_attach = attach
+AttachmentAlert_attach = attach,bijlage
```

Comment:
Cause this Zimlet only works for 1 language at the time. My users have Zimbra set to the default language english, and still expect to be warned if the missed an attachment while typing a dutch email.

Maybe I should fork this zimlet and make a 'Configurable version'...